torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -103,23 +103,15 @@ std::vector<at::Tensor> constructTensors(
   if (!qdataArg.has_value()) {
     for (const auto i : c10::irange(buf_data_vec.size())) {
       auto options = at::TensorOptions()
-                         // NOLINTNEXTLINE
                          .dtype(buf_dtypes_vec[i])
                          .layout(at::kStrided)
                          .device(at::kCPU) // TODO: support GPUs too
                          .memory_format(deduce_memory_format(
-                             // NOLINTNEXTLINE
-                             buf_strides_vec[i],
-                             // NOLINTNEXTLINE
-                             buf_dims_vec[i]))
+                             buf_strides_vec[i], buf_dims_vec[i]))
                          .requires_grad(false);
       auto tensor = at::from_blob(
```

Comment:
FYI, @cyyever this is bad code smell, and we should probably emplace the tensors from factory ctor in the future so they can be better elided.

Reply:
Given  the current state of JIT code and new alternatives, we may not need to change them so thoroughly.